### PR TITLE
Minor cleanup in teams and authz services

### PR DESCRIPTION
**Pull request by msorens**
### :nut_and_bolt: Description

Purely cosmetic cleanup:
* Consolidated use of literal "(unassigned)" into exactly one use per service.
* Lots of indentation cleanup for readability in postgres tests for teams.
* Removed duplicate import in one file.

### :+1: Definition of Done

Everything still works.

### :white_check_mark: Checklist

- [ ] Necessary tests added/updated?
- [ ] Necessary docs added/updated?
- [x] Code actually executed?
- [x] Vetting performed (unit tests, lint, etc.)?

### CODE REVIEW

File: components/authz-service/server/v2/policy_test.go [hunk 1]
```diff
@@ -22,7 +22,6 @@ import (
 
 	api_v2 "github.com/chef/automate/api/interservice/authz/v2"
 	constants_v1 "github.com/chef/automate/components/authz-service/constants/v1"
-	constants "github.com/chef/automate/components/authz-service/constants/v2"
```
Comment (msorens) [inline components/authz-service/server/v2/policy_test.go]: Removed duplicate import

File: components/teams-service/constants/constants.go [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package constants
```
Comment (msorens) [inline components/teams-service/constants/constants.go]: Reduce use of literal "(unassigned)" to just one instance in teams-service, just like in other services.

File: components/teams-service/storage/postgres/postgres.go [hunk 3]
```diff
@@ -111,9 +111,9 @@ func (p *postgres) getTeam(ctx context.Context, q querier, teamID uuid.UUID) (st
 	var t storage.Team
 	err := q.QueryRowContext(ctx,
 		`SELECT t.id, t.name, t.description, t.projects, t.updated_at, t.created_at
```
Comment (msorens) [inline components/teams-service/storage/postgres/postgres.go]: Changes in this file are all out ease of readability.

File: components/teams-service/storage/postgres/postgres.go [hunk 4]
```diff
@@ -212,7 +212,9 @@ func (p *postgres) GetTeams(ctx context.Context) ([]storage.Team, error) {
 	var teams []storage.Team
 	// TODO eventually these should be ordered
```
Comment (srenatus) [inline components/teams-service/storage/postgres/postgres.go]: Drive-by: is this still accurate? Do we have a card for it?

Comment (tylercloke) [reply components/teams-service/storage/postgres/postgres.go]: Don't think our queries are sorted. Not sure if we have a card or not.
